oas_docs/makefile

```diff
@@ -14,12 +14,36 @@
 # permission is obtained from Elasticsearch B.V.
 
 .PHONY: api-docs
-api-docs: ## Generate kibana.serverless.yaml
-	@npx @redocly/cli join "kibana.info.serverless.yaml" "../x-pack/plugins/observability_solution/apm/docs/openapi/apm.yaml" "../x-pack/plugins/actions/docs/openapi/bundled_serverless.yaml" "../src/plugins/data_views/docs/openapi/bundled.yaml" "../x-pack/plugins/ml/common/openapi/ml_apis_serverless.yaml" "../packages/core/saved-objects/docs/openapi/bundled_serverless.yaml" "../x-pack/plugins/observability_solution/slo/docs/openapi/slo/bundled.yaml" -o "kibana.serverless.yaml" --prefix-components-with-info-prop title
+api-docs: ## Generate kibana.serverless.yaml and kibana.yaml
```

Comment:
Will we be adding `oas_docs/bundle.json` and `oas_docs/bundle.serverless.json` in a future iteration? Is there a blocker to do so now?

Reply:
I have another PR to put up after this one. Basically need to add the missing tag to the /status API then refresh the bundles so it shows up nicely when joined with the rest.